testdata/cheats/Parse.t.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity >=0.8.0;
+
+import "forge-std/Test.sol";
+import "./Cheats.sol";
+
+contract ParseTest is Test {
+    Cheats constant cheats = Cheats(HEVM_ADDRESS);
+
+    function testParseBytes() public {
+        string memory stringBytes = "7109709ECfa91a80626fF3989D68f67F5b1DD12D";
+        bytes memory testBytes = hex"7109709ECfa91a80626fF3989D68f67F5b1DD12D";
+        assertEq(testBytes, cheats.parseBytes(stringBytes));
+    }
+
+    function testParseAddress() public {
+        string memory stringAddress = "0x7109709ECfa91a80626fF3989D68f67F5b1DD12D";
+        address testAddress = 0x7109709ECfa91a80626fF3989D68f67F5b1DD12D;
+        assertEq(testAddress, cheats.parseAddress(stringAddress));
```

Comment:
Should we also add a test that this works without the 0x prefix?

Reply:
good idea, added for bytes too